recipe/meta.yaml

```diff
@@ -37,20 +37,17 @@ test:
     - pep8
   imports:
     - cf_units
-  # commands:
-    # re-enable the tests when
-    # https://github.com/SciTools/cf-units/pull/116 is merged and released.
-    # - cd cf_units && python -m pytest tests
+  commands:
+    - cd cf_units && python -m pytest tests
 
 about:
-  home: http://www.scitools.org.uk/iris
-  license: LGPL-3.0
+  home: https://scitools.org.uk/cf-units/docs/latest/
+  license: LGPLv3
```

Comment:
@pelson this is not a big deal and you don't have to revert it but we are trying to consolidate the licenses strings around the [SPDX](https://spdx.org/licenses/LGPL-3.0.html) standard.